main.go

```diff
@@ -44,13 +42,11 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
-	var ociRegistryPort string
-	var ociRegistryServicceName string
+	var ociRegistryAddr string
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	flag.StringVar(&ociRegistryPort, "oci-registry-port", "5000", "The port the oci registry binds to.")
-	flag.StringVar(&ociRegistryServicceName, "oci-registry-service-name", "registry.ocm-system.svc.cluster.local", "Registry ervice")
+	flag.StringVar(&ociRegistryAddr, "oci-registry-addr", ":5000", "The address of the OCI registry.")
```

Comment:
I think you still need `address` and `port`. Our registry listen on `5000`, but we can change this as well as the service name.

Reply:
@souleb Wouldn't we just pass in a full address with the port included?